x-pack/plugins/ml/public/maps/layer_selector.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { Component } from 'react';
+
+import { EuiComboBox, EuiFormRow, EuiComboBoxOptionOption } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { MlAnomalyLayers } from './util';
+
+interface Props {
+  onChange: (typicalActual: MlAnomalyLayers) => void;
+  typicalActual: MlAnomalyLayers;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface State {}
+
+export class LayerSelector extends Component<Props, State> {
+  private _isMounted: boolean = false;
+
+  state: State = {};
+
+  componentDidMount(): void {
+    this._isMounted = true;
+  }
+
+  componentWillUnmount() {
+    this._isMounted = false;
+  }
+
+  onSelect = (selectedOptions: Array<EuiComboBoxOptionOption<string>>) => {
+    const typicalActual: MlAnomalyLayers = selectedOptions[0].value! as
+      | 'typical'
+      | 'actual'
+      | 'connected';
```

Comment:
I assume this shows a line from the typical position to the actual? If so, I'm not sure `connected` is the most intuitive label. Can't think of an alternative though!

Reply:
It is - I also couldn't think of anything more intuitive but am super open to suggestions 🙏